src/Raven.Studio/typescript/widgets/virtualGrid/columns/nodeTagColumn.ts

```diff
@@ -1,21 +1,30 @@
 ﻿/// <reference path="../../../../typings/tsd.d.ts"/>
 
 import virtualGridController = require("widgets/virtualGrid/virtualGridController");
-import hyperlinkColumn = require("./hyperlinkColumn");
+import hyperlinkColumn = require("widgets/virtualGrid/columns/hyperlinkColumn");
 
-class nodeTagColumn<T extends { nodeTag: string, database: string, noData: boolean }> extends hyperlinkColumn<T> {
+class nodeTagColumn<T extends { nodeTag: string, database: string, noData: boolean, isCommonItem: boolean }> extends hyperlinkColumn<T> {
+
+    hrefDestination: string;
     
-    constructor(gridController: virtualGridController<any>, hrefProvider: (item: T) => { url: string; openInNewTab: boolean }) {
+    constructor(gridController: virtualGridController<any>, hrefProvider: (item: T) => { url: string; openInNewTab: boolean }, hrefDestination: string) {
```

Comment:
I think new parameter should be called target, and be available add extra (and optional) field in hrefProvider. Currently we require this param, we assume that link need to go to some view + it can't be used for common item.

Reply:
also then we can remove isCommonItem from line number 6.

Reply:
provider will either provide data or no.

Reply:
done

Reply:
it is like target element on html it is optional.

Reply:
ok - made optional